public/data/imlakshay08.json

```diff
@@ -13,6 +13,16 @@
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/imLakshay08",
       "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/imlakshay08/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Mail me on",
```

Comment:
What do you think about this?
```suggestion
      "name": "Contact me via mail",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
I think it's a personal preference @Panquesito7, so it's good!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good